src/sentry/uptime/subscriptions/tasks.py

```diff
@@ -96,6 +99,8 @@ def uptime_subscription_to_check_config(
         "request_method": subscription.method,
         "request_headers": headers,
         "trace_sampling": subscription.trace_sampling,
+        "active_regions": [r.region_slug for r in subscription.regions.all()],
+        "region_schedule_mode": _CHECKCONFIGREGIONSCHEDULEMODE_ROUND_ROBIN,
```

Comment:
@evanpurkhiser I realised I defined this in the schema... Do you think it's better to use this or to define out own str enum?

Reply:
Personally I kind of prefer our own string enum since it’s easier to read and should correctly type check against this